tests/Unit/ControlSystem/ControlErrors/Test_Shape.cpp

```diff
@@ -64,10 +64,11 @@ void test_shape_control_error() {
   MAKE_GENERATOR(generator);
   domain::FunctionsOfTime::register_derived_with_charm();
 
-  const std::array<double, 3> origin{{0.0, 0.0, 0.0}};
+  const tnsr::I<double, 3, Frame::Grid> origin{0.};
+  const std::array<double, 3> origin_array{{0., 0., 0.}};
   const double ah_radius = 1.5;
   const double initial_time = 0.0;
-  Strahlkorper fake_ah{10, 10, origin};
+  Strahlkorper fake_ah{10, 10, origin_array};
```

Comment:
I think you can use `make_array` from `Utilities/MakeArray.hpp` here and in other places in this PR. `make_array<double, 3>(origin)` should do it.